src/screens/ParadeMapScreen/Map.test.js

```diff
@@ -1,18 +1,31 @@
 // @flow
-import { shallow } from "enzyme";
 import React from "react";
+import MapView, { Marker } from "react-native-maps";
+import { shallow } from "enzyme";
 import { TouchableWithoutFeedback } from "react-native";
-import MapView from "react-native-maps";
 import Permissions from "react-native-permissions";
 import Map, { checkLocationPermission, requestLocationPermission } from "./Map";
+import { generateEvent, sampleOne } from "../../data/__test-data";
 import * as position from "../../lib/position";
 import {
   region as paradeRegion,
   route,
   terminals
 } from "../../constants/parade-coordinates";
 
-const render = props => shallow(<Map {...props} />);
+const render = props =>
+  shallow(
+    <Map
+      {...props}
+      stages={[stages]}
+      addSavedEvent={() => {}}
+      removeSavedEvent={() => {}}
+      onPress={() => {}}
+      savedEvents={new Set()}
+    />
+  );
+
+const stages = sampleOne(generateEvent, { seed: 5728 });
```

Comment:
This is one `stage`, isn't it? 🙂

Reply:
lol, yes. I will change

Reply:
It caught me a few lines above where it is used as `stages={[stages]}`. I thought: why do we an array in an array? 😄

Reply:
Hm, you did change it. Just not in this file. It's fine, though. 😄

Reply:
Oh dear me! I have changed it here now too 😅